code/_globalvars/lists/objects.dm

```diff
@@ -9,6 +9,7 @@ GLOBAL_LIST_INIT(mechas_list, list())				//list of all mechs. Used by hostile mo
 GLOBAL_LIST_INIT(spacepods_list, list())				//list of all space pods. Used by hostile mobs target tracking.
 GLOBAL_LIST_INIT(joblist, list())					//list of all jobstypes, minus borg and AI
 GLOBAL_LIST_INIT(airlocks, list())					//list of all airlocks
+GLOBAL_LIST_EMPTY(alarmdisplay)				        //list of all machines or programs that can display station alerts
```

Comment:
Whats the reason for having this and `GLOB.alert_consoles`. Is `alert_consoles` a thing just for `for(x in alert_consoles)` speed?

Reply:
probably not needed now that I centralized the alarms into a SS, but it was absolutely needed in the beginning of this PR.